code/modules/reagents/reagent_containers/food/drinks.dm

```diff
@@ -1233,6 +1233,20 @@
 		light(user,I)
 		update_brightness(user)
 		return
+	else if(istype(I, /obj/item/weapon/reagent_containers/food/snacks/donut))
```

Comment:
Wouldn't it make more sense to make this into a drip proc for the donut itself and call that from here?

That's more maintainable if anyone else wants to make other things dippable or, if someone is looking for dip code, they'll find it in donut instead of drink